store/postgres/src/relational_queries.rs

```diff
@@ -213,6 +219,24 @@ impl<'a> QueryFragment<Pg> for QueryValue<'a> {
                         out.push_sql("[]");
                         Ok(())
                     }
+                    // TSVector will only be in a Value::List() for inserts so "to_tsvector" can always be used here
+                    ColumnType::TSVector((language, _)) => {
+                        let mut iter = values.iter().peekable();
+                        out.push_sql("(");
+                        while let Some(value) = iter.next() {
```

Comment:
Everywhere else in the file where we need to generate a list of things with a delimiter in between, we use the idiom
```rust
   for (i, thing) in things.iter().enumerate() {
     if i > 0 {
         // output delimiter
     }
     // output thing
   }
```

This code should follow that pattern.